### PR TITLE
QDEL CLEANUP 2 THE RECKONING

### PR DESCRIPTION
**Pull request by gbasood**
Sood's first code cleanup PR #6859 round 2
I probably missed something in here

This does a lot of things on top of changing del's to qdel's that don't matter including:
 - Deleting codeblocks that have long since been commented out
 - Changing del's in comment blocks to qdel's

### CODE REVIEW

File: code/WorkInProgress/Cael_Aislinn/Supermatter/Laser2.dm [hunk 1]
```diff
@@ -98,7 +98,7 @@
 
 /obj/beam/e_beam/Bump(atom/Obstacle)
 	Obstacle.laser_act(src)
-	del(src)
+	qdel(src)
```
Comment (N3X15) [inline code/WorkInProgress/Cael_Aislinn/Supermatter/Laser2.dm:101]: This is all commented out yo

File: code/modules/admin/verbs/antag_madness.dm [hunk 2]
```diff
@@ -387,8 +387,10 @@ client/proc/antag_madness(var/mob/M in mob_list)
 			M.equip_to_slot_or_del(pack, slot_in_backpack)
 			to_chat(M, "Your previous belongings have been stored in your backpack.")
 
-			del(M.wear_suit)
-			del(M.head)
+			M.wear_suit = null
+			M.head = null
+			qdel(M.wear_suit)
+			qdel(M.head)
```
Comment (N3X15) [inline code/modules/admin/verbs/antag_madness.dm]: why are you passing qdel null references?

File: code/modules/admin/verbs/randomverbs.dm [hunk 3]
```diff
@@ -846,7 +846,7 @@ Traitors and the like can also be revived with the previous role mostly intact.
 		message_admins("<span class='warning'>[usr.client.ckey] has banned [M.ckey].\nReason: [reason]\nThis is a permanent ban.</span>")
 		world.Export("http://216.38.134.132/adminlog.php?type=ban&key=[usr.client.key]&key2=[M.key]&msg=[html_decode(reason)]&time=perma&server=[replacetext(config.server_name, "#", "")]")
 		del(M.client)
-		del(M)
+		qdel(M)
```
Comment (N3X15) [inline code/modules/admin/verbs/randomverbs.dm:849]: Also commented out

File: code/modules/assembly/infrared.dm [hunk 4]
```diff
@@ -23,7 +23,7 @@
 		"On" = "on;number"
 	)
 
-	New() del(src)
+	New() qdel(src) // Why is this even
```
Comment (N3X15) [inline code/modules/assembly/infrared.dm]: new line plx

File: code/modules/awaymissions/maploader/swapmaps.dm [hunk 5]
```diff
@@ -295,10 +296,11 @@ swapmap
 					else
 						defarea.contents+=T
 					// clear the turf
-					for(var/obj/O in T) del(O)
+					for(var/obj/O in T) qdel(O)
```
Comment (N3X15) [inline code/modules/awaymissions/maploader/swapmaps.dm]: new line

File: code/modules/mob/living/carbon/alien/special/facehugger.dm [hunk 6]
```diff
@@ -118,7 +118,7 @@ var/const/MAX_ACTIVE_TIME = 400
 	// 		if(stat != DEAD)
 	// 			carr << "You pick up a facehugger"
 	// 			carr.facehuggers += 1
-	// 			del(src)
+	// 			qdel(src)
```
Comment (N3X15) [inline code/modules/mob/living/carbon/alien/special/facehugger.dm:121]: commented

File: code/modules/mob/living/simple_animal/hostile/giant_spider/effects.dm [hunk 7]
```diff
@@ -149,7 +149,7 @@
 /obj/effect/spider/spiderling/proc/die()
 	visible_message("<span class='alert'>[src] dies!</span>")
 	new /obj/effect/decal/cleanable/spiderling_remains(src.loc)
-	del(src)
+	qdel(src)
```
Comment (N3X15) [inline code/modules/mob/living/simple_animal/hostile/giant_spider/effects.dm:152]: commented

File: code/modules/power/switch.dm [hunk 8]
```diff
@@ -81,4 +81,4 @@
 		icon_state = icon_state_off
 		for(var/obj/structure/cable/C in src.loc)
 			del(C)
-*/
\ No newline at end of file
+*/
```
Comment (N3X15) [inline code/modules/power/switch.dm]: why

File: code/modules/research/mechanic/flatpack.dm [hunk 9]
```diff
@@ -255,11 +255,11 @@
 /datum/construction/flatpack_unpack/spawn_result(mob/user as mob)
 	var/obj/structure/closet/crate/flatpack/FP = holder
 	if(!istype(FP))
-		del(src)
+		qdel(src)
 		return
 	else
 		FP.Finalize()
-		del(src)
+		qdel(src)
```
Comment (N3X15) [inline code/modules/research/mechanic/flatpack.dm:262]: commented

**Review by N3X15 — CHANGES_REQUESTED** [review]

File: code/modules/admin/verbs/antag_madness.dm (hunk 2 above)
Comment (gbasood) [reply code/modules/admin/verbs/antag_madness.dm]: because I got the order wrong

File: code/modules/admin/verbs/randomverbs.dm (hunk 3 above)
Comment (gbasood) [reply code/modules/admin/verbs/randomverbs.dm:849]: this one was intentional, the file itself is used

File: code/modules/mob/living/carbon/alien/special/facehugger.dm (hunk 6 above)
Comment (gbasood) [reply code/modules/mob/living/carbon/alien/special/facehugger.dm:121]: intentional

File: code/modules/mob/living/simple_animal/hostile/giant_spider/effects.dm (hunk 7 above)
Comment (gbasood) [reply code/modules/mob/living/simple_animal/hostile/giant_spider/effects.dm:152]: intentional

File: code/modules/power/switch.dm (hunk 8 above)
Comment (gbasood) [reply code/modules/power/switch.dm]: that one wasn't me

File: code/modules/research/mechanic/flatpack.dm (hunk 9 above)
Comment (gbasood) [reply code/modules/research/mechanic/flatpack.dm:262]: intentional

File: code/WorkInProgress/Cael_Aislinn/Supermatter/Laser2.dm (hunk 1 above)
Comment (gbasood) [reply code/WorkInProgress/Cael_Aislinn/Supermatter/Laser2.dm:101]: this one was intentional, the file itself is used

**Review by N3X15 — APPROVED** [review]

**Pull request merged**